tests/old/config/test_config.py

```diff
@@ -120,10 +120,6 @@ def test_get_config_form_overview(self):
         response = self.client_manager.get('/config/')
         self.assertRedirects(response=response, expected_url='/config/general/',
                              status_code=302, target_status_code=200)
-        bad_client = Client()
-        response = bad_client.get('/config/', follow=True)
-        self.assertRedirects(response=response, expected_url='/login/?next=/config/general/',
```

Comment:
This test tests that an unauthorized client is not abled to see the config page. The second element (that he is redirected to the login page) does not need to be tested (here you are right). You can test an 302 HTTP status code if you like.